postman/202110-milmob-4/definition.js

```diff
@@ -0,0 +1,27 @@
+const currentTime = require('../utils/currentTime');
+let environments = require('./environment');
+
+let description = `
+This is a collection that represents the Prime's flow for the October Milmob 4
+Demo. The following Prime Requests are performed in this collection.
+
+- Request: Update shipment with a Reweigh
+  - Postman Request 1: Prime gets notified a move has been created
+  - Postman Request 2: Prime updates shipment with additional move details
+  - Postman Request 3: Prime performs a reweigh
+- Request: Prime gets notified that TOO approved move
+  - Postman Request 4: Prime creates payment request to get paid
+- Request: Prime submits a Proof of Service for Payment
+  - Postman Request 5: Prime uploads proof of service documents
```

Comment:
Just curious why the starting point wasn't the Sept Slice demo? The steps are nearly identical to that one as far as I can tell.

Reply:
I didn't realize both the postman collections were for reweighs. I was OOO for the September demo and didn't know it was about reweighs too. I probably should have used that one.

Reply:
Oh! and because the September one had sit extensions and the august one didn't.